EIPS/eip-7675.md

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
```suggestion
---
eip: 7675
```
Looks like you changed the filename but didn't add the assigned EIP number.

Reply:
Thanks @abcoathup !